src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,12 +22,21 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
+            requireNonNull(args);
+            ArgumentMultimap argMultimap =
+                    ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ID);
+            if (argMultimap.getValue(PREFIX_ID).isPresent()) {
```

Comment:
Does this account for the case where we want to reject a user who keys in both name and id?